test/e2e/angular/font-setting-test-ng.js

```diff
@@ -279,14 +279,14 @@
           element(by.css("#font1 .mce-btn[aria-label='Font Family']")).click();
           element(by.css("#mceu_634-text")).click();
 
-          expect(element(by.css("#font1 .text")).getCssValue("font-family")).to.eventually.equal("Roboto, sans-serif");
+          expect(element(by.css("#font1 .text")).getCssValue("font-family")).to.eventually.equal('\'Pinyon Script\', sans-serif');
         });
 
         it("should update font family for preview text if family name has spaces and numbers", function() {
           element(by.css("#font1 .mce-btn[aria-label='Font Family']")).click();
           element(by.css("#mceu_694-text")).click();
 
-          expect(element(by.css("#font1 .text")).getCssValue("font-family")).to.eventually.equal("'Slabo 27px', sans-serif");
+          expect(element(by.css("#font1 .text")).getCssValue("font-family")).to.eventually.equal("Risque, sans-serif");
```

Comment:
@rodrigopavezi This test is to check that fonts with spaces and numbers works. You'll need to find the new element to select that corresponds to "Slabo 27px" to preserve the integrity of the test.